src/mscorlib/shared/System/Numerics/Vector.cs

```diff
@@ -763,6 +767,37 @@ private Vector(ref Register existingRegister)
         {
             this.register = existingRegister;
         }
+
+#if netcoreapp
+        /// <summary>
+        /// Constructs a vector from the given span. The span must contain at least Vector'T.Count elements.
+        /// </summary>
+        public Vector(Span<T> values)
+            : this()
+        {
+            if ((typeof(T) == typeof(Byte))
```

Comment:
What about T == char?

Reply:
Vector doesn't support `char` because it isn't a number type.

Reply:
> Vector doesn't support `char` because it isn't a number type.

Ah I see. Tyvm.

Reply:
The type names `Byte` etc. come from existing template generation code [here](https://github.com/WinCPP/coreclr/blob/71d5215a2c007f7fe9c944bec47f0ed0fc70fefd/src/mscorlib/shared/System/Numerics/GenerationConfig.ttinclude#L9-L14). I did see VS suggesting me to use `byte`, etc. instead, but then I didn't purposely do that cleanup as it is not really a part of this PR.

Reply:
That's fair. It doesn't follow our coding guidelines, but like you mentioned, this should be fixed separately. Thanks!